manifests/e/Exsion365-B-V/ExsionTime/2.0.45321/Exsion365-B-V.ExsionTime.installer.yaml

```
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.4.3 $debug=NVS0.CRLF.5-1-26100-2161.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Exsion365-B-V.ExsionTime
+PackageVersion: 2.0.45321
+InstallerLocale: en-US
+InstallerType: inno
+InstallModes:
+- silent
+InstallerSwitches:
+  Silent: /SILENT
+  SilentWithProgress: /SILENT
+UpgradeBehavior: uninstallPrevious
+ReleaseDate: 2025-01-22
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/Exsion365-B-V/ExsionTimeVersions/blob/a32575415f6c226ffca95fed6e49d895f5b5b671/2.0.45321/ExsionTimeInstall.exe
+  InstallerSha256: 43F4699442AE8BF319877126FDFDE59EE793ACF12AABC97929190A95D344BF5B
```

```suggestion
 InstallerSha256: 49DEF0ED81E3725CD91B54246404BFF56FD82372A101BB42736353D867EA9993
```

(Automated response - build 965.)